GitUI/ScriptsEngine/IScriptHostControl.cs

```diff
@@ -7,5 +7,34 @@ public interface IScriptHostControl
         GitRevision? GetLatestSelectedRevision();
         IReadOnlyList<GitRevision> GetSelectedRevisions();
         Point GetQuickItemSelectorLocation();
+
+        void GoToRef(string? refName, bool showNoRevisionMsg, bool toggleSelection = false);
+
+        bool IsRevisionGrid { get; }
```

Comment:
```suggestion
        bool HasSelectedRevision { get; }
```

Reply:
It's also used to check for the availability for `GoToRef`. But I can change the name if you think it's still appropriate under that condition.

Reply:
> It's also used to check for the availability for `GoToRef`.

I missed this. But I think the check can be removed there when changing to `BrowseGoToRef`.